packages/vue-components/src/PageNavButton.vue

```diff
@@ -0,0 +1,45 @@
+<template>
+  <overlay
+    v-if="showPageNav"
+    :type="'pageNav'"
+    :src="src"
+    :fragment="'page-nav'"
+    :has-identifier="hasIdentifier"
+  />
+</template>
+
+<script>
+import $ from './utils/NodeList';
+
+export default {
+  data() {
+    return {
+      hasPageNav: false,
+      hasIdentifier: false,
+      show: false,
+      src: '',
+    };
+  },
+  computed: {
+    showPageNav() {
+      return this.show && this.hasPageNav;
```

Comment:
should it have `|| this.hasIdentifier` as well?

Reply:
Hmm actually this was implemented before but this will cause the page/site nav button to show up even if they are not present on the page. This happens when there is an identifier, for example, `id=page-nav` but the page does not have any headers, which will display an empty page nav menu when opened.

Reply:
ahh I see. Thanks for clarifying!